Resources/Textures/Objects/Weapons/Grenades/foamdart.rsi/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
```

Comment:
Are you sure about the license? I think goonStation's license is a little more serious.

Reply:
Another good catch! is CC-BY-NC-SA-3.0, which I'm fairly certain is fine since we're not selling the game.

Reply:
I don't think I need to change the others since they're just made by me based on the poster.

Reply:
Double checked and the new licenses are correct for goon & tg respectively.

Reply:
👍🏻